recipes-samples/images/rpb-minimal.bb

```diff
@@ -0,0 +1,10 @@
+SUMMARY = "Minimal image"
```

Comment:
Nitpick: images usually have 'image' in the name, 'bitbake rpb-minimal' could be confusing to users.